modules/sdk-coin-sei/src/tsei.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Testnet Sei
+ *
+ * @format
```

Comment:
any use of this? 🤔

Reply:
Not sure what is the use. It is added in all Testnet coin files. Hence added it.